app/models/piece.rb

```diff
@@ -68,11 +68,24 @@ def diagonal?(new_x, new_y, old_x, old_y)
     absolute_x == absolute_y ? true : false
   end
 
-  def piece_here?(x, y)
-    if Piece.where("x_coordinate = ? AND y_coordinate = ?", x, y) == []
+  def piece_here?(x, y, game)
+    if Piece.where(game_id: game_id, x_coordinate: x, y_coordinate: y) == []
       return false
     else
       return true
     end
   end
+
+
+  def move_to(x, y)
+    if piece_here?(x, y, self.game_id)
+      other_piece = Piece.where(game_id: game_id, x_coordinate: x, y_coordinate: y).first
+      if other_piece.white != self.white
+        other_piece.update_attributes(x_coordinate: 0, y_coordinate: 0, taken: true)
```

Comment:
What if we refactor this line out to a separate method, like: 'capture!(other_piece)'. It will be a one-liner and the `move_to!` method could almost be read as plain english :)